### ISSUE TITLE
Form Behaviors

### ISSUE THREAD

**Issue by naizy** [ajency/ajency.marionette#23]
Success or Error messages are currently just default strings. 
instead of defining them in the behavior's defaults, can pass the string in the function showSuccessMessage or showRequestFailerMessage.

eg. @showSuccessMessage "Organization details saved successfully"

This way we can show the error message returned by the server instead of default error text

**Comment by surajajency** [comment]
@naizy : Yes. I was thinking the same. Please create a pull request and contribute to it.
